app/_locales/en/messages.json

```diff
@@ -3052,6 +3058,12 @@
   "testFaucet": {
     "message": "Test Faucet"
   },
+  "theme": {
+    "message": "Theme"
+  },
+  "themeDescription": {
+    "message": "Switch to $1 mode"
```

Comment:
We need a `description` property here to describe what `$1` means.

Reply:
Actually, seeing this in practice, I think we can provide a more generic message:

"Choose your preferred MetaMask theme."

Reply:
updated 😁